stack/rest/src/test/java/org/apache/usergrid/rest/test/resource2point0/endpoints/EntityEndpoint.java

```diff
@@ -30,7 +30,12 @@
 
 import javax.ws.rs.core.MediaType;
 
-public class EntityEndpoint extends NamedResource {
+public class EntityEndpoint extends NamedResource<EntityEndpoint> {
+
+    @Override
+    protected EntityEndpoint getThis() {
+        return null;
```

Comment:
I'm not entirely clear why we want to return null here. Could you add a comment , just seems strange when looking at the others returning themselves.